recipe/meta.yaml

```diff
@@ -40,6 +40,7 @@ requirements:
     - scikit-learn >=0.17
     - spacy >=1.7.0,<2.0.0
     - unidecode >=0.04.19
+    - mwparserfromhell>=0.4.4
```

Comment:
You can remove this package as it's neither listed in textacy's setup or available in conda.

And you can add `tqdm>=4.11.1`.